pandas/tests/io/test_fsspec.py

```diff
@@ -62,6 +71,16 @@ def test_to_csv(cleared_fs):
     tm.assert_frame_equal(df1, df2)
 
 
+def test_csv_options(fsspectest):
```

Comment:
We'll probably want to parametrize these (see `pandas/tests/io/test_common.py` for examples)